### PR TITLE
DS-946 Remove duplicate editor config and update in openy_editor

### PR DESCRIPTION
**Pull request by froboy**
https://yusa.atlassian.net/browse/DS-946
related to https://github.com/open-y-subprojects/openy_features/pull/65

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use 9.x-2.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

- [x] Wait for https://github.com/open-y-subprojects/openy_features/pull/65 to be merged
- [ ] Test this PR
- [ ] Log in to the update and profile sites
- [ ] Check /admin/config/content/formats/manage/full_html on both
- [ ] Observe 1) the toolbar shows up properly, 2) the toolbar is the same on both.

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by podarok** [comment]
image

**Comment by podarok** [comment]
@froboy Upgrade path is broken now
```sh
11:01:41         ">  [error]  Configuration <em class=\"placeholder\">editor.editor.full_html</em> depends on the <em class=\"placeholder\">CKEditor 5 - Font Plugin (Text Color, Background Color)</em> module that will not be installed after import. ",
11:01:41         ">  [error]  Update failed: openy_node_session_update_8015 ",
11:01:41         ">  [notice] Update started: scheduler_update_8207",
11:01:41         ">  [notice] No update required.",
11:01:41         ">  [notice] Update completed: scheduler_update_8207",
11:01:41         ">  [notice] Update started: captcha_update_8907",
11:01:41         ">  [notice] Update completed: captcha_update_8907",
11:01:41         ">  [notice] Update started: image_captcha_update_9003",
11:01:41         ">  [notice] Update completed: image_captcha_update_9003",
11:01:41         ">  [notice] Update started: media_directories_ui_update_9002",
11:01:41         ">  [notice] Update completed: media_directories_ui_update_9002",
11:01:41         ">  [notice] Update started: openy_update_8095",
11:01:41         ">  [notice] Update completed: openy_update_8095",
11:01:41         ">  [notice] Update started: openy_activity_finder_update_8405",
11:01:41         ">  [error]  Configuration <em class=\"placeholder\">editor.editor.full_html</em> depends on the <em class=\"placeholder\">CKEditor 5 - Font Plugin (Text Color, Background Color)</em> module that will not be installed after import. ",
```

Because of module should be enabled before config imported

**Comment by froboy** [comment]
@podarok I'm investigating, but is it possible for the upgrade path checker to return an error to the comments on github if it fails? I feel like the silent failures make it easy to miss.

**Comment by podarok** [comment]
> @podarok I'm investigating, but is it possible for the upgrade path checker to return an error to the comments on github if it fails? I feel like the silent failures make it easy to miss.

It is possible, it was working back in a couple of years ago, but GitHub changes its API multiple times since then and even got acquired by Microsoft... )))

File: openy.install [hunk 1]
```diff
@@ -1226,19 +1227,25 @@ function openy_update_8091() {
...
+    ]);
 }
 
 /**
  * Enable ckeditor5 module.
  */
 function openy_update_8093() {
-  \Drupal::service('module_installer')->install(['ckeditor5']);
+    // Redo the above update in case folks missed it since we're rewriting history.
+    openy_update_8092();
```
Comment (froboy) [inline openy.install:1248]:
```suggestion
  // Redo the above update in case folks missed it since we're rewriting history.
  openy_update_8092();
```

**Comment by podarok** [comment]
profile site
image

**Comment by froboy** [comment]
Looks like this is good. Thanks @podarok !

**Pull request merged**